lib/intelipost/fluent_interface.rb

```diff
@@ -1,5 +1,22 @@
 module Intelipost
   module FluentInterface
+
+    def self.included(cls)
+      cls.extend(ClassMethods)
```

Comment:
`class`, but `class` is a reserved word. The most common is to use `klass` instead.